gradle/wrapper/gradle-wrapper.properties

```diff
@@ -1,5 +1,5 @@
 distributionBase=GRADLE_USER_HOME
 distributionPath=wrapper/dists
-distributionUrl=https\://services.gradle.org/distributions/gradle-6.2.1-bin.zip
+distributionUrl=https\://services.gradle.org/distributions/gradle-6.4-all.zip
```

Comment:
Do you see any particular reason to move from `bin` to `all`?

Reply:
Oops! For some reason I thought it was using `all` instead of `bin`. Fixed https://github.com/tfcporciuncula/flow-preferences/pull/5/commits/35b3714702a5a2ff36cdda9f225c573f4b37640a